std/range/package.d

```diff
@@ -7250,7 +7250,7 @@ if (isInputRange!Range)
     }
 
     // Assertion only.
-    private void dbgVerifySorted()
+    private auto dbgVerifySorted()
```

Comment:
Yeah, this compiles. There is another reason why this isn't working. I agree with @atilaneves.

```D
struct S(T)
{
    void foo() {}
}

void main() nothrow
{
    S!int s;
    s.foo();
}
```